actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -176,6 +176,8 @@ def use_cookies_with_metadata
   #   Default is +false+.
   # * <tt>:httponly</tt> - Whether this cookie is accessible via scripting or
   #   only HTTP. Defaults to +false+.
+  # * <tt>:same_site</tt> - Whether this cookie is should be restricted to a first-party or same-site context.
+  #   Defaults to +:lax+.
```

Comment:
I feel like if a user does not know about the `SameSite` attribute already, then they might not understand this option.  On the other hand, I think explaining the `SameSite` attribute is beyond the scope of this documentation.

Perhaps explicitly mentioning the attribute will point them in the right direction if they want to learn more?

```suggestion
  # * <tt>:same_site</tt> - The value of the +SameSite+ cookie attribute, which
  #   determines how this cookie should be restricted in cross-site contexts.
  #   Possible values are +nil+, +:none+, +:lax+, and +:strict+. Defaults to
  #   +:lax+.
```

Note that Rack actually [accepts more values](https://github.com/rack/rack/blob/ed308ff555ae0aeff465044be92bc6bcbc3ddaaa/lib/rack/utils.rb#L305-L316), but I think it is fine to say we only accept a subset.

Reply:
@jonathanhefner 

thank you for making sure I checked the suggestions, but I think it's simple and good.
However, I thought that the same can be said for `httponly`,` secure`, etc. when the user does not know about the option.
I felt that only the `same_site` added this time has more information than the existing options. Is that okay?
If it works, I'll fix the commit.

Reply:
> I felt that only the `same_site` added this time has more information than the existing options. Is that okay?

Yes, I think it is okay.  I think `httponly` and ` secure` are more straightforward, so less explanation is necessary.  Also, `httponly` and ` secure` are boolean, so possible values are not necessary.  ("Whether" =~ "yes or no".)

Reply:
I agree. Certainly `sami_site` is more difficult than` httponly`, `secure`, etc.